utils/view-options.ts

```diff
@@ -0,0 +1,8 @@
+export enum ViewOptions {
+  table = "table",
+  grid = "grid",
+}
+
+export const DEFAULT_VIEW_OPTIONS = [ViewOptions.table];
+
+export const ALL_VIEW_OPTIONS = [ViewOptions.table, ViewOptions.grid];
```

Comment:
Now we have consistent view option values between models and the atomic DataViewToggle